cfe_internal/enterprise/federation/federation.cf

```diff
@@ -192,6 +192,7 @@ bundle agent federation_manage_files
       string => "$(cfengine_enterprise_federation:config.transport_user)";
     "login" data => parsejson('{"login":"$(cfengine_enterprise_federation:config.login)"}');
     "feeder_username" data => parsejson('{"feeder_username":"$(cfengine_enterprise_federation:config.transport_user)"}');
+    "feeder" data => parsejson('{"feeder": "$(sys.key_digest)"}');
```

Comment:
@aleksandrychev @vpodzime If we use hostkey then it seems like we should require a hostkey value in the UI that goes to the API to manage remote hubs. Is this practical?

Reply:
Well, we need **some** unique identifier one way or another, right?

Reply:
@craigcomstock yes, but currently we don't require a name on the UI, I use API to get it automatically, I can rewrite logic to get hostkey as well.